mqtt/mqtt.js

```diff
@@ -1,3 +1,8 @@
+// Copyright (c) 2015-2019
```

Comment:
@darkodraskovic @blokovi please add copyright header of this type to Elm files.

Reply:
Resolved in the last commit.